main/src/library/Graph.flix

```diff
@@ -78,4 +72,21 @@ namespace Graph {
     pub def isConnected(src: {src :: t}, dst: {dst :: t}, g: m[(t, t)]): Bool with Foldable[m], Boxable[t] =
         reachable(src.src, g) |> Set.exists(x -> dst.dst == x)
 
+    ///
+    /// Returns the strongly connected components of the directed graph `g`.
+    ///
+    pub def stronglyConnectedComponents(g: m[(t, t)]): List[Set[t]] with Foldable[m], Boxable[t] =
```

Comment:
Should the return type be `Set[Set[t]]` for consistency?

Reply:
I'd say so yeah